packages/ateam-cli/cmd/agents-stop_agentStop.go

```
@@ -252,6 +252,9 @@ var agentsStopAgentStopCmd = &cobra.Command{
 
 		if apiErr != nil {
 			// defer poolSelfRelease runs on exit — pool slot is always released
+			if strings.Contains(apiErr.Error(), "WIP_LIMIT_EXCEEDED") {
+				return fmt.Errorf("%w\nThe target stage is at WIP capacity. Retry with --advance=false to log work and release the claim without advancing, then send ALERT to Hannibal to redispatch when capacity opens.", apiErr)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)